src/wasm-lib/kcl/tests/artifact_graph_sketch_on_face_etc/artifact_graph_flowchart.snap.md

```diff
@@ -124,14 +124,14 @@ flowchart LR
   26 --- 27
   26 --- 28
   26 --- 29
-  26 x--> 30
+  26 --- 30
   26 --- 31
   26 --- 32
   26 --- 33
   26 --- 34
   26 --- 35
   26 --- 36
-  30 --- 37
+  30 <--x 37
```

Comment:
Instead of the path of the sketch-on-face 37 being on a plane, it's now correctly on the end-cap. End-caps don't currently track all the things on them, so the edge is only from the path to the end-cap.

Reply:
Nice catch, yes I think this is why I couldn't sketch on end caps.

Reply:
Fixed in 18540642748f425f10153b5699be610f0ce26044.